timecop.gemspec

```diff
@@ -36,7 +36,6 @@ Gem::Specification.new do |s|
   s.homepage = %q{https://github.com/travisjeffery/timecop}
   s.rdoc_options = ["--charset=UTF-8"]
   s.require_paths = ["lib"]
-  s.rubyforge_project = %q{timecop}
```

Comment:
NOTE: Gem::Specification#rubyforge_project= is deprecated with no replacement. It will be removed on or after 2019-12-01.
Gem::Specification#rubyforge_project= called from /Users/nipe0324/project/contrib/timecop/timecop.gemspec:39.